src/snovault/elasticsearch/indexer.py

```diff
@@ -228,6 +229,12 @@ def index(request):
                     snapshot_id = connection.execute('SELECT pg_export_snapshot();').scalar()
 
     if invalidated and not dry_run:
+        invalid = []
+        for uuid in invalidated:
+            if len(invalid) < 100:
```

Comment:
Must remove before merging I think.

Reply:
yep,  I'd flag it myself but I can't review my own pr.